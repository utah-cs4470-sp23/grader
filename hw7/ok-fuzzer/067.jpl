write image (array[a : (- (sum[a : (sum[a : (- 282), b : (sum[a : argnum, b : argnum] a), c : 48] a)] (if true then 397 else argnum))), b : argnum] (if (array[c : a, d : (a * argnum), e : (225 % b)] (37.6 <= 49.5))[(if false then (- b) else a), argnum, (a * (b * 92))] then (if (array[c : a, d : b] true)[(sum[c : b, d : 397, e : a] a), (if true then argnum else a)] then {4.71, 34.24, (- 44.18), (sum[c : b] 6.1)} else {(sum[c : argnum] 42.68), (64.61 / 57.38), 9.76, 25.38}) else {(sum[c : (if true then b else a), d : (if true then 443 else a), e : (sum[c : b, d : a, e : 930] 320)] 13.08), (array[c : 465] 8.66)[(b * argnum)], 45.29, (array[c : b, d : argnum, e : argnum] 58.99)[b, (- b), argnum]})) to "a.png"

let a[b, c, d] = {argnum, {(array[a : 593, b : argnum, c : argnum] args)}, ((- 22.98) != (sum[a : argnum] 39.26))}{1}{0}
time let e[f, g] = {(array[e : (if false then d else c), f : b] args[394])}{0}

let h = false
write image {a, false, (array[i : argnum, j : (- d)] {61.73, (sum[k : j, l : g] 57.77), (sum[k : g, l : i] 70.62), (sum[k : 568, l : j, m : f] 14.39)})}{2} to "i.png"

let i = e
let {{j, k[l, m]}} = {(if (! (! (argnum == g))) then {{a, i}, d, f}{0} else {{a, e}, {{}}, argnum}{0})}
let n = false
let {} = {[(if false then {} else {}), {}]}{0}[(if n then b else (sum[o : (array[o : argnum, p : f, q : argnum] c)[(if n then g else g), f, b], p : ((sum[o : argnum, p : argnum, q : c] 368) / d), q : (array[o : b] g)[l]] o))]
time write image (array[o : b, p : f] {(47.45 % 97.57), (array[q : argnum, r : o] 62.95)[d, d], (- 2.57), {g, 41.51, n}{1}}) to "o.png"


let {o[p, q, r], {s, {t, u[v, w, x], y}, z[A]}} = {(array[o : l, p : c, q : 525] (array[r : e[g, c]] e)), {{m, {c, j, l}, [true, h, true]}, argnum}{0}}
write image (array[B : f, C : g] (array[D : C, E : t] {12.22, 88.56, (- 54.75), 33.23}))[r, (if (if n then ((! n) == h) else (m == v)) then A else t)] to "B.png"

