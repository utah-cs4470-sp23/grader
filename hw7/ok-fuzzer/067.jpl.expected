(ShowCmd
 (IfExpr
  (BoolType)
  (TupleIndexExpr
   (BoolType)
   (TupleLiteralExpr
    (TupleType (BoolType))
    (UnopExpr (BoolType) ! (UnopExpr (BoolType) ! (FalseExpr (BoolType)))))
   0)
  (BinopExpr
   (BoolType)
   (TupleIndexExpr
    (FloatType)
    (TupleLiteralExpr
     (TupleType (FloatType))
     (SumLoopExpr
      (FloatType)
      a
      (VarExpr (IntType) argnum)
      (FloatExpr (FloatType) 87)))
    0)
   <=
   (TupleIndexExpr
    (FloatType)
    (TupleLiteralExpr
     (TupleType (FloatType) (IntType))
     (BinopExpr
      (FloatType)
      (FloatExpr (FloatType) 98)
      +
      (FloatExpr (FloatType) 67))
     (IntExpr (IntType) 435))
    0))
  (TupleIndexExpr
   (BoolType)
   (TupleLiteralExpr
    (TupleType (BoolType) (IntType) (BoolType))
    (BinopExpr
     (BoolType)
     (VarExpr (IntType) argnum)
     !=
     (IntExpr (IntType) 887))
    (ArrayIndexExpr
     (IntType)
     (VarExpr (ArrayType (IntType) 1) args)
     (VarExpr (IntType) argnum))
    (UnopExpr (BoolType) ! (UnopExpr (BoolType) ! (FalseExpr (BoolType)))))
   2)))
(WriteCmd
 (IfExpr
  (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 2)
  (UnopExpr
   (BoolType)
   !
   (TupleIndexExpr
    (BoolType)
    (TupleLiteralExpr
     (TupleType (IntType) (BoolType) (TupleType (IntType) (IntType)))
     (SumLoopExpr
      (IntType)
      a
      (VarExpr (IntType) argnum)
      (VarExpr (IntType) argnum))
     (BinopExpr
      (BoolType)
      (VarExpr (IntType) argnum)
      !=
      (VarExpr (IntType) argnum))
     (TupleLiteralExpr
      (TupleType (IntType) (IntType))
      (IntExpr (IntType) 406)
      (VarExpr (IntType) argnum)))
    1))
  (ArrayLoopExpr
   (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 2)
   a
   (IntExpr (IntType) 715)
   b
   (BinopExpr
    (IntType)
    (VarExpr (IntType) argnum)
    %
    (BinopExpr
     (IntType)
     (VarExpr (IntType) argnum)
     /
     (BinopExpr
      (IntType)
      (VarExpr (IntType) argnum)
      +
      (VarExpr (IntType) argnum))))
   (TupleLiteralExpr
    (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
    (FloatExpr (FloatType) 7)
    (UnopExpr
     (FloatType)
     -
     (SumLoopExpr
      (FloatType)
      c
      (IntExpr (IntType) 179)
      d
      (VarExpr (IntType) a)
      e
      (VarExpr (IntType) b)
      (FloatExpr (FloatType) 68)))
    (UnopExpr (FloatType) - (FloatExpr (FloatType) 36))
    (FloatExpr (FloatType) 27)))
  (ArrayLoopExpr
   (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 2)
   a
   (BinopExpr
    (IntType)
    (UnopExpr
     (IntType)
     -
     (SumLoopExpr
      (IntType)
      a
      (IntExpr (IntType) 338)
      b
      (IntExpr (IntType) 700)
      c
      (IntExpr (IntType) 93)
      (VarExpr (IntType) c)))
    -
    (IntExpr (IntType) 357))
   b
   (BinopExpr
    (IntType)
    (BinopExpr (IntType) (IntExpr (IntType) 229) - (VarExpr (IntType) argnum))
    %
    (SumLoopExpr
     (IntType)
     a
     (VarExpr (IntType) argnum)
     b
     (VarExpr (IntType) argnum)
     (VarExpr (IntType) argnum)))
   (TupleLiteralExpr
    (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
    (SumLoopExpr
     (FloatType)
     c
     (UnopExpr (IntType) - (IntExpr (IntType) 168))
     (UnopExpr (FloatType) - (FloatExpr (FloatType) 16)))
    (SumLoopExpr
     (FloatType)
     c
     (VarExpr (IntType) b)
     (UnopExpr (FloatType) - (FloatExpr (FloatType) 37)))
    (BinopExpr
     (FloatType)
     (FloatExpr (FloatType) 82)
     +
     (FloatExpr (FloatType) 42))
    (FloatExpr (FloatType) 46))))
 "a.png")
(ShowCmd (VarExpr (IntType) argnum))
(LetCmd
 (TupleLValue
  (TupleLValue
   (TupleLValue
    (ArgLValue (ArrayArgument a b c d))
    (TupleLValue (ArgLValue (ArrayArgument e f)) (ArgLValue (VarArgument g)))
    (ArgLValue (ArrayArgument h i j k)))
   (ArgLValue (VarArgument l))))
 (TupleLiteralExpr
  (TupleType
   (TupleType
    (TupleType
     (ArrayType (IntType) 3)
     (TupleType (ArrayType (BoolType) 1) (ArrayType (BoolType) 1))
     (ArrayType (FloatType) 3))
    (TupleType (FloatType) (TupleType) (BoolType))))
  (TupleLiteralExpr
   (TupleType
    (TupleType
     (ArrayType (IntType) 3)
     (TupleType (ArrayType (BoolType) 1) (ArrayType (BoolType) 1))
     (ArrayType (FloatType) 3))
    (TupleType (FloatType) (TupleType) (BoolType)))
   (TupleLiteralExpr
    (TupleType
     (ArrayType (IntType) 3)
     (TupleType (ArrayType (BoolType) 1) (ArrayType (BoolType) 1))
     (ArrayType (FloatType) 3))
    (ArrayLoopExpr
     (ArrayType (IntType) 3)
     a
     (IntExpr (IntType) 136)
     b
     (BinopExpr
      (IntType)
      (VarExpr (IntType) argnum)
      %
      (VarExpr (IntType) argnum))
     c
     (IntExpr (IntType) 741)
     (VarExpr (IntType) a))
    (TupleLiteralExpr
     (TupleType (ArrayType (BoolType) 1) (ArrayType (BoolType) 1))
     (ArrayLoopExpr
      (ArrayType (BoolType) 1)
      a
      (VarExpr (IntType) argnum)
      (FalseExpr (BoolType)))
     (ArrayLiteralExpr
      (ArrayType (BoolType) 1)
      (FalseExpr (BoolType))
      (TrueExpr (BoolType))))
    (ArrayLoopExpr
     (ArrayType (FloatType) 3)
     a
     (ArrayIndexExpr
      (IntType)
      (VarExpr (ArrayType (IntType) 1) args)
      (IntExpr (IntType) 661))
     b
     (BinopExpr
      (IntType)
      (VarExpr (IntType) argnum)
      *
      (VarExpr (IntType) argnum))
     c
     (VarExpr (IntType) argnum)
     (SumLoopExpr
      (FloatType)
      d
      (VarExpr (IntType) c)
      e
      (VarExpr (IntType) b)
      (FloatExpr (FloatType) 32))))
   (TupleLiteralExpr
    (TupleType (FloatType) (TupleType) (BoolType))
    (SumLoopExpr
     (FloatType)
     a
     (IntExpr (IntType) 659)
     b
     (IntExpr (IntType) 356)
     (FloatExpr (FloatType) 67))
    (TupleLiteralExpr (TupleType))
    (ArrayIndexExpr
     (BoolType)
     (ArrayLoopExpr
      (ArrayType (BoolType) 2)
      a
      (VarExpr (IntType) argnum)
      b
      (IntExpr (IntType) 769)
      (TrueExpr (BoolType)))
     (IfExpr
      (IntType)
      (FalseExpr (BoolType))
      (IntExpr (IntType) 76)
      (IntExpr (IntType) 64))
     (UnopExpr (IntType) - (VarExpr (IntType) argnum)))))))
(LetCmd (ArgLValue (ArrayArgument m n)) (VarExpr (ArrayType (BoolType) 1) e))
(LetCmd
 (ArgLValue (ArrayArgument o p q))
 (ArrayLoopExpr
  (ArrayType (ArrayType (IntType) 1) 2)
  o
  (VarExpr (IntType) i)
  p
  (VarExpr (IntType) argnum)
  (VarExpr (ArrayType (IntType) 1) args)))
(LetCmd
 (TupleLValue)
 (TupleIndexExpr
  (TupleType)
  (IfExpr
   (TupleType (IntType) (IntType) (TupleType))
   (IfExpr
    (BoolType)
    (ArrayIndexExpr
     (BoolType)
     (VarExpr (ArrayType (BoolType) 1) m)
     (VarExpr (IntType) d))
    (BinopExpr
     (BoolType)
     (BinopExpr (BoolType) (VarExpr (IntType) n) > (VarExpr (IntType) p))
     &&
     (UnopExpr (BoolType) ! (FalseExpr (BoolType))))
    (BinopExpr
     (BoolType)
     (FloatExpr (FloatType) 29)
     >=
     (FloatExpr (FloatType) 95)))
   (IfExpr
    (TupleType (IntType) (IntType) (TupleType))
    (IfExpr
     (BoolType)
     (FalseExpr (BoolType))
     (ArrayIndexExpr
      (BoolType)
      (VarExpr (ArrayType (BoolType) 1) m)
      (VarExpr (IntType) f))
     (FalseExpr (BoolType)))
    (IfExpr
     (TupleType (IntType) (IntType) (TupleType))
     (FalseExpr (BoolType))
     (TupleLiteralExpr
      (TupleType (IntType) (IntType) (TupleType))
      (VarExpr (IntType) d)
      (VarExpr (IntType) q)
      (TupleLiteralExpr (TupleType)))
     (TupleLiteralExpr
      (TupleType (IntType) (IntType) (TupleType))
      (VarExpr (IntType) j)
      (VarExpr (IntType) j)
      (TupleLiteralExpr (TupleType))))
    (IfExpr
     (TupleType (IntType) (IntType) (TupleType))
     (ArrayIndexExpr
      (BoolType)
      (VarExpr (ArrayType (BoolType) 1) m)
      (VarExpr (IntType) k))
     (TupleLiteralExpr
      (TupleType (IntType) (IntType) (TupleType))
      (VarExpr (IntType) n)
      (VarExpr (IntType) i)
      (TupleLiteralExpr (TupleType)))
     (TupleLiteralExpr
      (TupleType (IntType) (IntType) (TupleType))
      (VarExpr (IntType) q)
      (VarExpr (IntType) q)
      (TupleLiteralExpr (TupleType)))))
   (TupleIndexExpr
    (TupleType (IntType) (IntType) (TupleType))
    (TupleLiteralExpr
     (TupleType (TupleType (IntType) (IntType) (TupleType)))
     (TupleLiteralExpr
      (TupleType (IntType) (IntType) (TupleType))
      (VarExpr (IntType) k)
      (VarExpr (IntType) argnum)
      (TupleLiteralExpr (TupleType))))
    0))
  2))
(LetCmd
 (ArgLValue (VarArgument r))
 (BinopExpr (IntType) (VarExpr (IntType) j) + (VarExpr (IntType) b)))
(LetCmd (ArgLValue (VarArgument s)) (VarExpr (ArrayType (BoolType) 1) m))
(AssertCmd
 (ArrayIndexExpr
  (BoolType)
  (IfExpr
   (ArrayType (BoolType) 2)
   (BinopExpr
    (BoolType)
    (ArrayIndexExpr
     (IntType)
     (ArrayLoopExpr
      (ArrayType (IntType) 3)
      t
      (VarExpr (IntType) p)
      u
      (VarExpr (IntType) n)
      v
      (VarExpr (IntType) c)
      (VarExpr (IntType) f))
     (BinopExpr (IntType) (VarExpr (IntType) j) % (VarExpr (IntType) d))
     (VarExpr (IntType) j)
     (VarExpr (IntType) b))
    <=
    (IfExpr
     (IntType)
     (FalseExpr (BoolType))
     (VarExpr (IntType) d)
     (VarExpr (IntType) b)))
   (TupleIndexExpr
    (ArrayType (BoolType) 2)
    (TupleLiteralExpr
     (TupleType (ArrayType (IntType) 3) (ArrayType (BoolType) 2) (IntType))
     (VarExpr (ArrayType (IntType) 3) a)
     (ArrayLoopExpr
      (ArrayType (BoolType) 2)
      t
      (VarExpr (IntType) r)
      u
      (VarExpr (IntType) k)
      (TrueExpr (BoolType)))
     (VarExpr (IntType) argnum))
    1)
   (ArrayLoopExpr
    (ArrayType (BoolType) 2)
    t
    (VarExpr (IntType) f)
    u
    (VarExpr (IntType) c)
    (TupleIndexExpr
     (BoolType)
     (VarExpr (TupleType (FloatType) (TupleType) (BoolType)) l)
     2)))
  (IfExpr
   (IntType)
   (UnopExpr (BoolType) ! (TrueExpr (BoolType)))
   (UnopExpr (IntType) - (VarExpr (IntType) q))
   (VarExpr (IntType) d))
  (VarExpr (IntType) n))
 "t")
(ShowCmd (VarExpr (IntType) argnum))
(LetCmd
 (TupleLValue
  (TupleLValue (ArgLValue (VarArgument t)) (ArgLValue (VarArgument u)))
  (ArgLValue (ArrayArgument v w x y))
  (ArgLValue (VarArgument z)))
 (ArrayIndexExpr
  (TupleType
   (TupleType (IntType) (IntType))
   (ArrayType (FloatType) 3)
   (BoolType))
  (ArrayLoopExpr
   (ArrayType
    (TupleType
     (TupleType (IntType) (IntType))
     (ArrayType (FloatType) 3)
     (BoolType))
    1)
   t
   (IfExpr
    (IntType)
    (BinopExpr
     (BoolType)
     (BinopExpr
      (BoolType)
      (FloatExpr (FloatType) 62)
      >
      (FloatExpr (FloatType) 30))
     !=
     (BinopExpr (BoolType) (VarExpr (IntType) n) != (IntExpr (IntType) 631)))
    (VarExpr (IntType) p)
    (VarExpr (IntType) q))
   (TupleLiteralExpr
    (TupleType
     (TupleType (IntType) (IntType))
     (ArrayType (FloatType) 3)
     (BoolType))
    (TupleLiteralExpr
     (TupleType (IntType) (IntType))
     (VarExpr (IntType) d)
     (VarExpr (IntType) f))
    (ArrayIndexExpr
     (ArrayType (FloatType) 3)
     (ArrayLoopExpr
      (ArrayType (ArrayType (FloatType) 3) 3)
      u
      (VarExpr (IntType) r)
      v
      (VarExpr (IntType) r)
      w
      (VarExpr (IntType) r)
      (VarExpr (ArrayType (FloatType) 3) h))
     (VarExpr (IntType) d)
     (VarExpr (IntType) p)
     (IntExpr (IntType) 286))
    (BinopExpr
     (BoolType)
     (BinopExpr
      (BoolType)
      (FloatExpr (FloatType) 6)
      >=
      (FloatExpr (FloatType) 98))
     ==
     (TrueExpr (BoolType)))))
  (VarExpr (IntType) b)))
